microsoft-edge/visual-studio-code/elements-for-edge.md

```diff
@@ -0,0 +1,70 @@
+---
+description: How to use Elements for Microsoft Edge (Chromium) from VS Code
+title: Debug Microsoft Edge (Chromium) from VS Code
```

Comment:
I don't believe this is the right title in the metadata. The title is important because it renders as the title for the tab. I would suggest changing this to "Elements for Microsoft Edge".